plugin/trino-hive/src/test/java/io/trino/plugin/hive/containers/HiveMinioDataLake.java

```diff
@@ -42,6 +53,8 @@
     private final AtomicBoolean isStarted = new AtomicBoolean(false);
     private final AutoCloseableCloser closer = AutoCloseableCloser.create();
 
+    private Optional<AmazonS3> s3Client = Optional.empty();
```

Comment:
this should be thread-safe, as tests can run in parallel, so add `@GuardedBy("this")` and synchronize `getS3Client`